### DISCUSSION TITLE
Add promoted attributes to all notes via a default template?

### DISCUSSION THREAD

**Discussion by d-vyd** [TriliumNext/Trilium#1518] [Q&A]
Hi folks,

I'd like each note to have a NoteDate that is different from the date the note was created or modified. I think a template would allow me to do this, but I would have to create a relation to the template to inherit the attribute. That would take about the same time as adding NoteDate manually. Can I create a default template that all my notes automatically follow?

thank you
-d. vyd

**Comment by zadam** [comment]
Hi, you can assign the template relation to root note and make this relation inheritable. Then all notes will use this template since they are all descendants of root note.

**Reply by d-vyd** [reply]
ah! that is perfect. thank you.
